installing/installing_bare_metal/bare-metal-using-bare-metal-as-a-service.adoc

```diff
@@ -0,0 +1,28 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="bare-metal-using-bare-metal-as-a-service"]
+= Using bare metal as a service
+include::_attributes/common-attributes.adoc[]
+:context: bare-metal-using-bmaas
+
+toc::[]
+
+{product-title}'s Bare Metal as a Service (BMaaS) feature enables you to provision and manage bare-metal nodes using the Metal^3^ API. These nodes, external to the {product-title} cluster, can host workloads requiring direct hardware access, such as legacy applications or high-performance computing tasks. BMaaS provides the following capabilities:
```

Comment:
Suggestion: These nodes, external to the {product-title} cluster, can host workloads which may not be suitable for containerisation or virtualisation such as applications requiring direct hardware access, high-performance computing tasks or legacy applications.

Reply:
*-isation is me using UK/AU spelling, feel free to change to preference